lib/term.js

```diff
@@ -30,23 +30,23 @@ function literal (value, languageOrDatatype) {
   return term
 }
 
-function namedNode (value) {
+function namedNode (value, factory = rdf) {
```

Comment:
I don't think it makes sense to place a parameter on every method. You would always use `this.factory` IMO

Reply:
You mean not have a default value?

Reply:
I mean no parameter at all

```diff
-function namedNode (value, factory = rdf) {
+function namedNode (value) {

-return factory.namedNode(value)
+return this.factory.namedNode(value)
```

Reply:
But these functions are standalone. They don't have access to the context (`this`).

Reply:
aha, sorry, I missed your reply. thought it was on the Clownface object

then I agree with what you said before. the default seems redundant

Reply:
Yes, it's a standalone, but the factory argument should be always provided, so there should be no default value for it.

Reply:
That should be the case for all functions in `lib/term.js`.

Reply:
That means `term` has to be called like this in some cases: `term('test', undefined, undefined, factory)`. Should the factory be the first param? Or should we abandon the idea of having pure functions for this module and move them to a class?

Reply:
Let's keep them as pure functions.